SpeechResponder/HelpWindow.xaml.cs

```diff
@@ -17,17 +17,19 @@ public HelpWindow()
 
             // Read Markdown and convert it to HTML
             string markdown;
+            string blabla = "";
```

Comment:
Please use a meaningful variable name.
Edit: actually, why is this variable even here?

Reply:
oups, sorry, i forget to remove the line 20, a test i use sometimes and remove after